Samples/StorageProviders/StorageProviders/BaseJSONStorageProvider.cs

```diff
@@ -15,6 +17,13 @@ namespace Samples.StorageProviders
     /// </summary>
     public abstract class BaseJSONStorageProvider : IStorageProvider
```

Comment:
we have a class with the same full name in (test\TesterInternal\StorageTests\StorageProviders\BaseJSONStorageProvider.cs)[https://github.com/dotnet/orleans/blob/master/test/TesterInternal/StorageTests/StorageProviders/BaseJSONStorageProvider.cs] . We should probably keep them in sync.